internal/driver/controllerserver.go

```diff
@@ -93,7 +93,7 @@ func (cs *ControllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 	}
 
 	// Create the volume
-	vol, err := cs.createAndWaitForVolume(ctx, volName, sizeGB, req.GetParameters()[VolumeTags], sourceVolInfo, accessibilityRequirements)
+	vol, err := cs.createAndWaitForVolume(ctx, volName, req.GetParameters()[VolumeTags], req.GetParameters()[VolumeEncryption], sizeGB, sourceVolInfo, accessibilityRequirements)
```

Comment:
Nit: we can just pass req.GetParameters() here and handle getting the volumetags and volumEncryption in the func. That would be much cleaner approach

Reply:
Could you also set the volumeContext to include the encryption is set to true? This will be useful to double check in the controllerpublishvolume() grpc func!

Reply:
Yup, working on these changes.